test/functional/apps/management/_test_huge_fields.js

```diff
@@ -0,0 +1,21 @@
+import expect from 'expect.js';
+
+export default function ({ getService, getPageObjects }) {
+  const esArchiver = getService('esArchiver');
+  const PageObjects = getPageObjects(['common', 'home', 'settings']);
+
+  describe('test large number of fields', function () {
+    const EXPECTED_FIELD_COUNT = 10006;
+    before(async function () {
+      await esArchiver.loadIfNeeded('large_fields');
```

Comment:
let's also have an `after` that unloads this archive

Reply:
addressed.